perf/allocs.jl

```diff
@@ -149,65 +150,33 @@ function plot_allocs(
     filter!(x -> x.bytes ≠ 0, allocs_per_case)
     filter!(x -> !compile_pkg(x.filename, x.linenumber), allocs_per_case)
 
-    for alloc in allocs_per_case
-        println(alloc)
-    end
-    println("Number of allocating sites: $(length(allocs_per_case))")
-    case_bytes = getproperty.(allocs_per_case, :bytes)[end:-1:1]
-    case_filename = getproperty.(allocs_per_case, :filename)[end:-1:1]
-    case_linenumber = getproperty.(allocs_per_case, :linenumber)[end:-1:1]
+    case_bytes = reverse(getproperty.(allocs_per_case, :bytes))
+    case_filename = reverse(getproperty.(allocs_per_case, :filename))
+    case_linenumber = reverse(getproperty.(allocs_per_case, :linenumber))
+
     all_bytes = Int[]
-    filenames = String[]
-    linenumbers = Int[]
     loc_ids = String[]
+
     for (bytes, filename, linenumber) in
         zip(case_bytes, case_filename, case_linenumber)
         compile_pkg(filename, linenumber) && continue # Skip loading module
-        loc_id = "$(filename_only(filename))" * "$linenumber"
+        loc_id = "$(filename_only(filename)):$(linenumber)"
         if !(bytes in all_bytes) && !(loc_id in loc_ids)
             push!(all_bytes, bytes)
-            push!(filenames, filename)
-            push!(linenumbers, linenumber)
             push!(loc_ids, loc_id)
             if length(all_bytes) ≥ n_unique_bytes
                 break
             end
         end
     end
-
-    all_bytes = all_bytes ./ 10^3
-    max_bytes = maximum(all_bytes)
-    @info "$case_name_args: $all_bytes"
-    xtick_name(filename, linenumber) = "$filename, line number: $linenumber"
-    markershape = (:square, :hexagon, :circle, :star, :utriangle, :dtriangle)
-    for (bytes, filename, linenumber) in zip(all_bytes, filenames, linenumbers)
-        label = xtick_name(filename_only(filename), linenumber)
-        Plots.plot!(
-            [0],
-            [bytes];
-            seriestype = :scatter,
-            label = label,
-            markershape = markershape[1],
-            markersize = 1 + bytes / max_bytes * 10,
-        )
-        markershape = (markershape[end], markershape[1:(end - 1)]...)
+    all_kbytes = map(b -> div(b, 10^3), all_bytes)
```

Comment:
```suggestion
    all_kbytes = map(b -> div(b, 1024), all_bytes) # convert to KiB
```
Maybe we should report with KiB, to be consistent with BenchmarkTools?